tools/ReleaseTool/PrepCommand.cs

```diff
@@ -103,30 +107,16 @@ public int Run()
                 }
 
                 // Create pull request
-                PullRequest pullRequest;
-                try
-                {
-                    pullRequest = gitHubClient.CreatePullRequest(gitHubRepo, branchName, options.DevBranch,
-                        string.Format(PullRequestTemplate, options.Version));
-                }
-                catch (Exception e)
-                {
-                    Console.WriteLine("WARNING: Creating Pull Request Failed. This might be because the pull " +
-                                      $"request already exists? Error: {e}");
-                    pullRequest = null;
-                }
+                var pullRequest = gitHubClient.CreatePullRequest(gitHubRepo, branchName, options.DevBranch,
+                    string.Format(PullRequestTemplate, options.Version));
 
-                Console.WriteLine(string.Empty);
-                Console.WriteLine($"Release hash: {gitClient.GetHeadCommit().Sha}");
-                if (pullRequest != null)
-                {
-                    Console.WriteLine(string.Empty);
-                    Console.WriteLine($"Pull request available: {pullRequest.HtmlUrl}");
-                }
+                Logger.Info("Successfully created release!");
+                Logger.Info("Release hash: {0}", gitClient.GetHeadCommit().Sha);
+                Logger.Info("Pull request available: {0}", pullRequest.HtmlUrl);
```

Comment:
Could open the URL automatically 😉 

https://stackoverflow.com/questions/4580263/how-to-open-in-default-browser-in-c-sharp

Reply:
Done 👍